libs/openblas/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=OpenBLAS
+PKG_SOURCE_VERSION:=e55ec82bb92338d09ecd77357da3fcdfac0a7902
+PKG_VERSION:=0.3.9-$(PKG_SOURCE_VERSION)
+PKG_RELEASE:=1
+#PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+#PKG_SOURCE_URL:=https://github.com/xianyi/OpenBLAS/archive/
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2020-04-19
+PKG_SOURCE_URL:=https://github.com/martin-frbg/OpenBLAS.git
+#PKG_HASH:=28cc19a6acbf636f5aab5f10b9a0dfe1
+PKG_BUILD_DIR:=$(BUILD_DIR)/OpenBLAS-$(PKG_VERSION)
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openblas
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=An optimized BLAS library
+  URL:=https://www.openblas.net/
+endef
+
+define Package/bridge/description
+  OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version
+endef
+# Package build instructions; invoke the GNU make tool to build our package
+define Build/Compile
```

Comment:
if you adjust the build flags via MAKE_FLAGS & MAKE_VARS, can you get rid of the explicit `Build/Compile` rule?
usually adding it explicitly is a good idea only if there quirks for building
but the default rules are pretty good